pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
@palewire Typically we reuse docstring on several places, eg for Series/DataFrame/Panel definitions, that's the reason of the use of `_shared_docs`.

But, @jreback, was just looking in this specific case, this is the only place where this docstring is used, so it is actually not really needed to put it in `_shared_docs` I think? (maybe a leftover from when the definitions where in multiple places)